CONTRIBUTING.md

```diff
@@ -54,6 +54,20 @@ see [ABI Policy Guide](sycl/doc/ABIPolicyGuide.md) for more information.
 - Run regression tests -
 [instructions](sycl/doc/GetStartedGuide.md#test-dpc-toolchain).
 
+**Guidelines for adding Clang FE tests**:
+- Use `sycl::` namespace instead of `cl::sycl::`
+
+- Include sycl mock headers as system headers.
+Example: `-internal-isystem %S/Inputs`
+`#include "sycl.hpp"`
+
+- Use SYCL functions for invoking kernels from the mock header `(single_task, parallel_for)`
```

Comment:
Looks good.   I think this one - using SYCL function - would be a very helpful example.   The class template with the sycl_kernel attribute was a very easy thing to write before, so having an example to use instead would come very handy.   Could you add that here?

Reply:
Did you mean to include an example like below?

#include "Inputs/sycl.hpp"
sycl::queue q;
q.submit([&](cl::sycl::handler &h) {
    h.**single_task**<class kernel_name>([]() { //code });
  });